cmake/configure/configure_1_mpi.cmake

```diff
@@ -30,7 +30,8 @@ MACRO(FEATURE_MPI_FIND_EXTERNAL var)
   IF(MPI_FOUND)
     SET(${var} TRUE)
 
-    IF(NOT MPI_HAVE_MPI_SEEK_SET)
+    # MPI_HAVE_MPI_SEEK_SET can not be verified during cross compiling
+    IF((NOT MPI_HAVE_MPI_SEEK_SET) AND (NOT CMAKE_CROSSCOMPILING))
```

Comment:
Is `CMAKE_CROSSCOMPILING` an actual cmake thing, or is this something that you set locally in your build scripts?

Reply:
It's a boolean set by cmake: see also

https://cmake.org/cmake/help/latest/variable/CMAKE_CROSSCOMPILING.html

Reply:
The variable `CMAKE_CROSSCOMPILING` will be set to true automatically when you define the CMake cross compiling toolchain using the command line option `-DCMAKE_TOOLCHAIN_FILE=file.cmake`